src/components/AddBookForm.js

```diff
@@ -1,14 +1,33 @@
+import { useDispatch, useSelector } from 'react-redux';
 import React from 'react';
+import { addBookAction } from '../redux/books/books';
 
-const AddBookForm = () => (
-  <section className="add-book-form">
-    <h2>ADD NEW BOOK</h2>
-    <form className="books-list">
-      <input type="text" placeholder="Book title" />
+const AddBookForm = () => {
+  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
 
-      <button type="button">ADD BOOK</button>
-    </form>
-  </section>
-);
+  const returnState = (e) => {
+    e.preventDefault();
+    const authorInput = e.target.querySelector('.authorInput');
+    const titleInput = e.target.querySelector('.titleInput');
```

Comment:
- [ ] To get the value from each input field, I will recommend you use the useState hook containing the data variables while the inputs will have a function (e.g `handleChange()`)that updates the local state every time they update using the onChange attribute on the input field.👍🏽